agent/connect/ca/provider_vault.go

```diff
@@ -38,6 +38,17 @@ func (v *VaultProvider) Configure(clusterId string, isRoot bool, rawConfig map[s
 	clientConf := &vaultapi.Config{
 		Address: config.Address,
 	}
+	err = clientConf.ConfigureTLS(&vaultapi.TLSConfig{
+		CACert:        config.CAFile,
+		CAPath:        config.CAPath,
+		ClientCert:    config.CertFile,
+		ClientKey:     config.KeyFile,
+		Insecure:      config.SkipValidate,
```

Comment:
Should this value `TLSSkipVerify` instead?

Reply:
absolutely! good catch!